netlify.toml

```diff
@@ -1,2 +1,4 @@
 [build]
-  ignore = "echo 0"
\ No newline at end of file
+  base = "client/flutter"
+  command = "chmod +x helper-scripts/flutter-netlify-build.sh && ./helper-scripts/flutter-netlify-build.sh"
```

Comment:
```suggestion
  command = "./helper-scripts/flutter-netlify-build.sh"
```

Reply:
This PR initially did this, but Netlify threw errors because the file wasn't executable.

Reply:
I think that was because the executable bit was not set in git.